generate-release-notes.sh

```diff
@@ -238,6 +238,20 @@ https://github.com/aws/amazon-ecs-ami/blob/main/CHANGELOG.md#$ami_version
             read ami_name_al2_kernel_5_10_arm agent_version_al2_kernel_5_10_arm docker_version_al2_kernel_5_10_arm source_ami_name_al2_kernel_5_10_arm <<<$(get_ami_details "/aws/service/ecs/optimized-ami/amazon-linux-2/kernel-5.10/arm64/recommended")
             add_ami_to_release_notes "#### ARM64 (Kernel 5.10)" "$ami_name_al2_kernel_5_10_arm" "$agent_version_al2_kernel_5_10_arm" "$docker_version_al2_kernel_5_10_arm" "$containerd_version" "$runc_version" "" "" "$source_ami_name_al2_kernel_5_10_arm" ""
         fi
+
+        # Include AL2 Neuron (Kernel 5.10) release notes if there was an al2kernel5dot10inf release
```

Comment:
nit: should this be inferentia instead of neuron?

Reply:
I think we should leave as Neuron, as this is what we refer it as on L214 and L216, as well as that is what is surfaced on this documentation page:
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html

Reply:
I think we should leave as Neuron, as this is what we refer it as on L214 and L216, and that is what is surfaced on this documentation page:
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html

Reply:
ah nevermind it's referred to as neuron for AL2 already, disregard

Reply:
yep, sounds good to me. thanks!